workos/audit_trail.py

```diff
@@ -48,21 +50,126 @@ def create_event(self, event, idempotency_key=None):
             idempotency_key (str) - An idempotency key
 
         Returns:
-            dict: Response from WorkOS
+            boolean: Returns True
         """
         if len(event.get("metadata", {})) > METADATA_LIMIT:
-            raise Exception(
+            raise ValueError(
                 "Number of metadata keys exceeds {}.".format(METADATA_LIMIT)
             )
 
         headers = {
             "idempotency-key": idempotency_key,
         }
 
-        return self.request_helper.request(
+        self.request_helper.request(
             EVENTS_PATH,
             method=REQUEST_METHOD_POST,
             params=event,
             headers=headers,
             token=workos.api_key,
         )
+
+        return True
```

Comment:
why return True instead of the response? looking at the other methods, it looks like the norm is to return the response, ex.`SSO.create_connection`?

Reply:
Don't think there are stories for this yet but we've been wanting to create `Resources` for everything. I also sorta don't like returning the raw response as well which was why I opted for this